lib/view_data/pg/handler.rb

```diff
@@ -135,8 +135,12 @@ def configure
         logger.info(tag: :data) { pkey_values.pretty_inspect }
       end
 
-      def double_quote(text)
-        "\"#{text}\""
+      def quote_columns(column)
+        session.connect.quote_ident(column.to_s)
+      end
```

Comment:
Seems like this method should be called `quote_column` (singular), since it only quotes a single column.

Reply:
makes sense. Updated.